packages/create-plugin/templates/common/.config/Dockerfile

```diff
@@ -1,7 +1,12 @@
-ARG grafana_version=latest
-ARG grafana_image=grafana-enterprise
+ARG ARCH=amd64
+ARG BASE_IMAGE=grafana/grafana-oss-${ARCH}:development-ubuntu
```

Comment:
This changes https://github.com/grafana/plugin-tools/pull/197. @mckn we still would like enterprise as the default?

Reply:
It's possible to maintain the base image as it was before with a few adjustments, we would have to select alpine or ubuntu to install the necessary packages (dlv, supervisord, etc.). The differences that I can see is that the development image suggested by Brian, provides node running in development mode and the possibility to build for an specific architecture. (maybe I'm missing other points)
Regarding debugging, capabilities, it's the same.

@briangann Do you think that it would be a significant loss to go with grafana-enterprise as a base image?

Reply:
Another idea would be to provide two totally separated environments, like staging and dev. The dev one we do based on the dev images and the staging we do based on grafana enterprise.

Reply:
@marefr we decided to use the non dev base images as it was before, therefore the Dockerfiles will still have the grafana enterprise as default.

Reply:
👍